components/list/list-item-checkbox-mixin.js

```diff
@@ -81,7 +81,8 @@ export const ListItemCheckboxMixin = superclass => class extends SkeletonMixin(L
 		super.connectedCallback();
 		if (this.selectable) {
 			if (!this.key) console.warn('ListItemCheckboxMixin requires a key.');
-			if (!this.label || this.label.length === 0) console.warn('ListItemCheckboxMixin requires a label.');
+		} else {
+			this._labelRequired = false;
```

Comment:
Since `ListItemCheckboxMixin` is unfortunately _always_ mixed into all list items... we need a way to disable the label validation if selection isn't enabled. This logic may need to move into the `ListItemMixin` if it becomes more complex (e.g. selection OR drag & drop requires a label).